pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -86,7 +92,14 @@ func (admitter *KubeVirtUpdateAdmitter) Admit(ar *admissionv1.AdmissionReview) *
 		results = append(results, validateInfraReplicas(newKV.Spec.Infra.Replicas)...)
 	}
 
-	return validating_webhooks.NewAdmissionResponse(results)
+	response := validating_webhooks.NewAdmissionResponse(results)
+
+	if toVerifyFeatureGates(&currKV.Spec, &newKV.Spec) {
```

Comment:
I don't think the name of this function is very straightforward, can you change it? 
Maybe something like `featureGatesChanged()` is more to the point.

Reply:
@acardace Sorry can you please give an example by what you mean?

Reply:
Nice, will do